opencog/spacetime/octomap/OctoMapValue.h

```diff
@@ -0,0 +1,71 @@
+#ifndef  _OPENCOG_OCTOMAP_VALUE_H_
+#define  _OPENCOG_OCTOMAP_VALUE_H_
+
+
+#include <opencog/atoms/value/Value.h>
+#include <opencog/atoms/value/FloatValue.h>
+#include <opencog/atoms/value/atom_types.h>
+#include <opencog/atomspace/AtomSpace.h>
+
+#include <opencog/atoms/value/ValueFactory.h>
+
+#include <typeindex>
+
+#include "OctoMapNode.h"
+#include "TimeOctomap.h"
+
+namespace opencog
+{
+/** \addtogroup grp_atomspace
+ *  @{
+ */
+
+// OctoValue holds 3d coordinates of an atom. When it is
+// set, it stores in the TimeOctoMap instance provided by
+// OctoMapNode.
```

Comment:
This comment is wrong. The OctoValue does NOT hold the 3d coordinates! The OctoMap holds the 3D coordinates. The OctoValue provides a way to obtain the current 3D coordinates from the OctoMap.